test/e2e_node/summary_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega/types"
 )
 
-const restartCount = 3
+const restartCount = 1
```

Comment:
this makes the container restart exactly once.  Is there a way to check the restart count?  I thought it was just encoded in the container name?